src/EnergyPlus/SolarCollectors.cc

```diff
@@ -709,10 +703,6 @@ namespace SolarCollectors {
             } // ICSNum
 
             if (ErrorsFound) ShowFatalError(state, "Errors in " + CurrentModuleObject + " input.");
-
-            if (state.dataSolarCollectors->NumOfCollectors > 0) {
-                state.dataSolarCollectors->CheckEquipName.dimension(state.dataSolarCollectors->NumOfCollectors, true);
```

Comment:
CheckEquipName was never actually being used for this object, so just deleted the array and this line of code that was allocating it.